### DISCUSSION TITLE
Possibility to import custom rules.txt files automatically

### DISCUSSION THREAD

**Discussion by torontal7** [uBlockOrigin/uBlock-issues#3109] [Ideas] [answered]
I use only live operating sytems. So, my question is: is it possible to import a custom rules.txt file automatically and this how can be done? By "automatically" I mean not manually, not using UBO control panel. Instead if there exist a rules.txt in ~/.config/chromium/Default/Extensions/cjpalhdlnbpafiamejdnhcphjbkeiagm/1.32.4_0/assets/ublock directory it would be automatically applied/imported without any user action. I already experienced that this does not work, I have to import manually my rules. So, if this is not possible currently my feature request is to be implemented.

**Comment by ghost** [comment, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Deploying-uBlock-Origin
